sql/python/sqlflow/template/instance_db_test_.py

```diff
@@ -0,0 +1,48 @@
+import os
```

Comment:
I am a little confused about why we need two test files for db.py.

Then I realized that is because .travis.yml called two shell scripts: scripts/test.sh and scripts/test_hive.sh.

However, I still feel that we can have only one test file, say, test_db.py, which contains

```python
class TestDB(TestCase):
   def test_mysql(self):
      ...
   def test_sqlite(self):
      ...
   def test_hivel(self):
      ...
```

where in each method, say, `test_mysql`, it judges the value of environment variable `SQLFLOW_TEST_DB`:

```python
def test_mysql(self):
    if os.environ['SQLFLOW_TEST_DB'] == "" OR os.environ['SQLFLOW_TEST_DB'] == "mysql:
        go_on_test
```